clients/osv/src/main/kotlin/Model.kt

```diff
@@ -25,6 +25,15 @@ import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
 import kotlinx.serialization.json.JsonObject
 
+// FIXME: Remove all JsonElement subtypes as property types from the model in favor of raw strings holding JSON.
```

Comment:
Can't you simply use a custom deserializer as outlined [here](https://github.com/Kotlin/kotlinx.serialization/issues/1058#issuecomment-983269638) that calls `encodeToString()` on the `JsonElement`? Performance should not be an issue here, as I do not expect these model element to be large.

Reply:
I've seen this comment and tried to write code based on it. I haven't figured it out though. As you say "simply", maybe you know how to use it to reach the goal. In particular, it was unclear what this `polymorphicSerlializer` variable is and how to modify `BaseClass` in that code snippet to make it compile. Do you have a hint?